bcda/models/fhir/alr/v1/coverage.go

```diff
@@ -15,28 +15,35 @@ func coverage(mbi string, keyValue []utils.KvPair, lastUpdated time.Time) *fhirm
 
 	coverage := &fhirmodels.Coverage{}
 	coverage.Id = &fhirdatatypes.Id{Value: mbi}
-	coverage.Meta = &fhirdatatypes.Meta{
-		Profile: []*fhirdatatypes.
-			Uri{{Value: "http://alr.cms.gov/ig/StructureDefinition/alr-Coverage"}},
-	}
 	coverage.Extension = []*fhirdatatypes.Extension{}
 	coverage.Beneficiary = &fhirdatatypes.Reference{
-		Reference: &fhirdatatypes.Reference_OrganizationId{
-			OrganizationId: &fhirdatatypes.ReferenceId{Value: mbi},
+		Reference: &fhirdatatypes.Reference_PatientId{
+			PatientId: &fhirdatatypes.ReferenceId{Value: mbi},
 		},
 	}
+	// coverage.Payor = []*fhirdatatypes.Reference{}
+	// payorRef := &fhirdatatypes.Reference{
+	// 	Reference: &fhirdatatypes.Reference_OrganizationId{
+	// 		OrganizationId: &fhirdatatypes.ReferenceId{Value: "example-org-id"},
+	// 	}}
+	// coverage.Payor = append(coverage.Payor, payorRef)
+
```

Comment:
Should we just delete these lines?

Reply:
Ah, waiting on feedback from Bill on these - will adjust accordingly in the next round!